python/ray/_private/node.py

```diff
@@ -270,7 +270,11 @@ def __init__(
 
         # Pick a GCS server port.
         if head:
-            gcs_server_port = os.getenv(ray_constants.GCS_PORT_ENVIRONMENT_VARIABLE)
+            from ray.autoscaler._private.constants import env_integer
```

Comment:
can you use the method under ray._private.ray_constants, not the autoscaler package? (not sure why it has the same method there)